src/domain/select_local_cameras.py

```diff
@@ -0,0 +1,126 @@
+"""Local cameras configuration module."""
+
+from PySide6.QtWidgets import QDialog, QLabel, QCheckBox, QLineEdit, QHBoxLayout, QPushButton, QDialogButtonBox
+from PySide6.QtGui import QFont
+from PySide6.QtCore import Qt
+from cv2_enumerate_cameras import enumerate_cameras
+import cv2
+
+from domain.camera import Camera
+from ui.ui_select_local_cameras import Ui_DialogSelectLocalCameras
+
+class DialogSelectLocalCameras(QDialog, Ui_DialogSelectLocalCameras):
+    """Class to select and configure local cameras (i.e., directly plugged to the WADAS node)."""
+    def __init__(self, cameras_list):
+        super(DialogSelectLocalCameras, self).__init__()
+        self.ui = Ui_DialogSelectLocalCameras()
+        self.ui.setupUi(self)
+        self.cameras_list = cameras_list
+
+        self.list_usb_cameras_in_ui()
+        self.initialize_detection_params()
+
+        # Slots
+        self.ui.buttonBox.accepted.connect(self.accept_and_close)
+
+    def list_usb_cameras_in_ui(self):
+        """Method to initialize cameras list."""
+
+        # Setting titles to columns
+        table_titles =["Enabled", "Index", "Name", "PID", "VID", "WADAS ID", "Test"]
+        boldFont=QFont()
+        boldFont.setBold(True)
+        row = col = 0
+        for title in table_titles:
+            label = QLabel(title)
+            label.setFont(boldFont)
+            self.ui.gridLayout_localCameras.addWidget(label, row, col)
+            col = col+1
+
+        row = 1
+        for camera_info in enumerate_cameras(cv2.CAP_MSMF): 
+            #TODO: check API preference for linux and implement it OS dependent.
+            camera_index = str(camera_info.index)
+          
+            # Camera selection check box
+            check_box = QCheckBox()
+            checkbox_obj_name = "checkBox_camera"+camera_index
+            check_box.setObjectName(checkbox_obj_name)
+            check_box.checkStateChanged.connect(self.validate_cameras_selection)
+            self.ui.gridLayout_localCameras.addWidget(check_box, row, 0)
+             # Camera index
+            label = QLabel(f'{camera_info.index}')
+            label.setObjectName("label_cameraIdx"+camera_index)
+            self.ui.gridLayout_localCameras.addWidget(label, row, 1)
+            # Camera name
+            label = QLabel(f'{camera_info.name}')
+            label.setObjectName("label_cameraName"+camera_index)
+            self.ui.gridLayout_localCameras.addWidget(label, row, 2)
+            # Camera PID
+            label = QLabel(f'{camera_info.pid}')
+            label.setObjectName("label_cameraPid"+camera_index)
+            self.ui.gridLayout_localCameras.addWidget(label, row, 3)
+            # Camera VID
+            label = QLabel(f'{camera_info.vid}')
```

Comment:
Better to use str() instead of f-string on those cases.

Reply:
what's the difference? I would like to uniform the approach...